src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -26,11 +26,24 @@
     private BlockNetworkUtil() {
     }
 
+    /**
+    * Get the side of a block that is rotated based on the defined side
+    *
+    * @param block - the block that is rotated
```

Comment:
Remove the dash between `block` and `the block....`
Just to be consistent with Javadocs in other modules